python/lsst/ts/mtdome/mock_llc/mock_motion/azimuth_motion.py

```diff
@@ -116,8 +128,23 @@ def __init__(
         self._commanded_motion_state = LlcMotionState.PARKED
         # Keep track of the motion state where we came from.
         self._current_motion_state = LlcMotionState.PARKED
+        # Keep track of being in error state or not.
+        self.motion_state_in_error = False
```

Comment:
Can you tell that you the current state is an error by examining self._current_motion_state, or any other atrributes? If so, I strongly suggest making this a property that you compute, rather than having to keep track of a separate attribute that can get out of sync?

I hope you can compute it. It seems weird that the dome can report a motion state of "moving" or "crawling" or whatever, and also be in an error state at the same time.

Reply:
The way the current position, velocity and motion_state are computed relies on the motion_state_in_error property. It probably can be computed but that would require rewriting the `get_position_velocity_and_motion_state` method which I prefer not to do at this point because it already is quite a complex method that required several changes to support the new commands.

Reply:
I'm not sure if I was sufficiently clear. If the the value of this attribute can be computed then I suggest making a property:
```
@property
def self.motion_state_in_error(self):
     #... compute the value and return it
```
With this change you would not have to change any code that *uses* the value. The advantage is that it avoids having to set an attribute that must be carefully tied to other attributes.

Reply:
i hope you will consider doing this before merging (if such a calculation is straightforward).